src/contrib/serializers/Akka.Serialization.TestKit/AkkaSerializationSpec.cs

```diff
@@ -394,7 +394,7 @@ public void CanSerializeSupervisor()
 
         //TODO: find out why this fails on build server
 
-        [Fact(Skip = "Fails on buildserver")]
```

Comment:
I dont know the status of these, enabling to see what the buildserver think nowadays